app/listings/[id]/edit/page.tsx

```
@@ -0,0 +1,232 @@
+          variant: "destructive",
+        });
+        router.push("/dashboard/listings");
+      } else {
+        const validation = listingSchema.safeParse(data);
+        if (validation.success) {
+          setListing(validation.data);
+        } else {
+          toast({
+            title: "Data Error",
+            description:
+              "The listing data is invalid. " + validation.error.message,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)